ios/Classes/NativeBridge/Handlers/ResourcesHandler.swift

```diff
@@ -170,23 +160,17 @@ class ResourcesHandler: FetchResourcesHandlerProtocol {
 
     func fetchResourceWithImplicitResource(_ path: String, parameters: [String: Any?], completion: @escaping FlutterResult) {
         Logger.log("fetchResourceWithImplicitResource", sender: self)
-        guard let _profile = ONGUserClient.sharedInstance().authenticatedUserProfile() else {
-            completion(SdkError(.authenticatedUserProfileIsNull))
+        guard let _ = ONGUserClient.sharedInstance().implicitlyAuthenticatedUserProfile() else {
+            completion(SdkError(.unauthenticatedImplicitly).flutterError())
             return
         }
 
         let newParameters = generateParameters(from: parameters, path: path)
         let scopes = newParameters["scope"] as? [String]
         
-        OneginiModuleSwift.sharedInstance.authenticateUserImplicitly(_profile.profileId, scopes: scopes) { (value, error) in
-            if (error == nil) {
-                OneginiModuleSwift.sharedInstance.resourceRequest(isImplicit: value, parameters: newParameters) { (_data, error) in
-                    if let data = _data {
-                        completion(data)
-                    } else {
-                        completion(error)
-                    }
-                }
+        OneginiModuleSwift.sharedInstance.resourceRequest(isImplicit: true, parameters: newParameters) { (_data, error) in
+            if let data = _data {
```

Comment:
How about, instead of this whole if/else just
```
(data, error)
+
completion(data ?? error)

Reply:
done